lms/templates/wiki/base.html

```diff
@@ -1,4 +1,5 @@
 {% extends "main_django.html" %}
+{% with online_help_token="wiki" %}
```

Comment:
@nasthagiri , I think the syntax is different here because the wiki page uses Django templating and the other pages we added links to use Mako templates. Is that correct?